java/sage/Channel.java

```diff
@@ -74,6 +76,14 @@
     }
     else
       logoImages = Pooler.EMPTY_INT_ARRAY;
+    if (ver >= 0x56)
+    {
+      int urlLen = in.readShort();
+      logoURL = new byte[urlLen];
```

Comment:
In the one arg constructor, set logoURL to be an empty byte array. This shouldn't matter, but it's a good idea for consistency.